pkg/volume/secret/secret.go

```diff
@@ -43,7 +43,8 @@ const (
 
 // secretPlugin implements the VolumePlugin interface.
 type secretPlugin struct {
-	host volume.VolumeHost
+	host          volume.VolumeHost
```

Comment:
Given that - do you think it's better to ad GetSecret method (and in the future GetConfigMap) to VolumeHost? Or leave it as here? I would vote for the above approach, but I don't have strong opinion.
@saad-ali @yujuhong @liggitt ?